var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -263,7 +266,7 @@ class Seacas(CMakePackage):
         depends_on("cgns@4.2.0:~mpi+scoping", when="~mpi")
 
     with when("+adios2"):
-        depends_on("adios2@master")
+        depends_on("adios2@2.10.1")
```

Comment:
Does this need to be a specific release or is any newer fine?
```suggestion
        depends_on("adios2@2.10.1:")
```

Reply:
It is the lastest release that thy currently have, so I'm not sure whether future releases will work.  I do know that there is currently a problem with there latest development version (that they are aware of). 

I'm fine assuming it will work with future, or waiting until they release and then testing...

Reply:
I am not familiar enough with these packages to know how tightly interwoven they are and how much breakage is typically involved in adios2 version upgrades, so I'll defer to your expertise and mark this as resolved.